assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -43,6 +43,8 @@ import {
 import paneId from './paneId';
 import MediaElement from './mediaElement';
 
+export const ROOT_MARGIN_TOP = 300;
```

Comment:
Should this be ROOT_MARGIN_BOTTOM?

Reply:
Yes, changing that to ROOT_MARGIN because we will ever use just one (bottom).